docs/ref/models/indexes.txt

```diff
@@ -32,6 +32,19 @@ references of :class:`Index` which includes the `index options`_.
 
 A list of the name of the fields on which the index is desired.
 
+By default the indexes are created with an ascending order for all the columns.
+To define an index with a descending order for a particular column, a hyphen
+(-) can be added before the field's name while defining the index.
+
+For example::
+
+    Index(fields=['headline', '-pub_date'], name='headline_pub_date_reverse_idx')
+
+The above index would be issue an SQL with ``(headline, pub_date DESC)``. This
+is supported only for PostgreSQL, Oracle and some versions of SQLite. If your
+database doesn't support defining ordering on index columns it will simply be
```

Comment:
PostgreSQL and Oracle always use the defined order.

From the [docs of SQLite](https://www.sqlite.org/lang_createindex.html)
> The sort order may or may not be ignored depending on the database file format, and in particular the schema format number

From the [docs of MySQL](http://dev.mysql.com/doc/refman/5.7/en/create-index.html)
> An index_col_name specification can end with ASC or DESC. These keywords are permitted for future extensions for specifying ascending or descending index value storage. Currently, they are parsed but ignored; index values are always stored in ascending order.


What type of test is required? Should I write version-specific tests for SQLite when it is ignoring and when not?